documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia: {
```

Comment:
I think this is in the wrong location, based on what's in the schema? It seems like it should be under `reporters`.